### PR TITLE
[EN/ZH] Add `Yuyuko's Hidden Cup Season 2` and Update `tournaments`

### PR DESCRIPTION
**Pull request by YumeMuzi**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

**Comment by TPGPL** [comment]
pretty sure wiki doesn't support `.svg` files like that, you should convert them to `.jpg` and add some background as it may not be well visible on the site

File: wiki/Tournaments/YHC/2/en.md [hunk 1]
```diff
@@ -0,0 +1,322 @@
+---
+tags:
+  - YHC S2
+  - YHC
+---
+
+# Yuyuko's Hidden Cup Season 2
+
+![YHC S2 Banner](img/banner.jpg)
```
Comment (TPGPL) [inline wiki/Tournaments/YHC/2/en.md:9]:
```suggestion
![YHC S2 banner](img/banner.jpg) 
```

File: wiki/Tournaments/YHC/2/en.md [hunk 2]
```diff
@@ -0,0 +1,322 @@
...
+tags:
+  - YHC S2
+  - YHC
+---
+
+# Yuyuko's Hidden Cup Season 2
+
+![YHC S2 Banner](img/banner.jpg) 
+
+The **Yuyuko's Hidden Cup Season 2** (***YHC S2***) was a team-based 3v3 double-elimination osu! gamemode tournament hosted by ::{ flag=CN }:: [SIyuyuko](https://osu.ppy.sh/users/9794030). It was the 2nd installment of the Yuyuko's Hidden Cup.
```
Comment (TPGPL) [inline wiki/Tournaments/YHC/2/en.md:11]:
```suggestion
The **Yuyuko's Hidden Cup Season 2** (***YHC S2***) was a team-based 3v3 double-elimination osu! tournament hosted by ::{ flag=CN }:: [SIyuyuko](https://osu.ppy.sh/users/9794030). It was the 2nd instalment of the Yuyuko's Hidden Cup.
```

File: wiki/Tournaments/YHC/2/en.md [hunk 3]
```diff
@@ -0,0 +1,322 @@
...
+
+## Tournament schedule
+
+| Event | Timestamp |
+| --: | :-- |
+| Registration phase | 2022-11-26/2022-12-25 |
+| Drawings | 2022-12-25 (20:00 UTC+08) |
+| Group stage | 2022-12-28/2023-1-8 |
+| Elimination stage 1 | 2023-1-11/2023-1-15 |
+| Elimination stage 2 | 2023-1-18/2023-2-5 |
```
Comment (TPGPL) [inline wiki/Tournaments/YHC/2/en.md:21]:
```suggestion
| Group stage | 2022-12-28/2023-01-08 |
| Elimination stage 1 | 2023-01-11/2023-01-15 |
| Elimination stage 2 | 2023-01-18/2023-02-05 |
```

File: wiki/Tournaments/YHC/2/en.md [hunk 4]
```diff
@@ -0,0 +1,322 @@
...
+## Links
+
+- [Livestream](https://live.bilibili.com/21549800)
+- [YHC Community QQ Group](https://jq.qq.com/?_wv=1027&k=4CVOhMz7)
+- [Challonge bracket](https://challonge.com/YHC_S2)
+- **[Statistics sheet](https://docs.qq.com/sheet/DUkhYRERQUWVTY2lE?tab=v89kx8)**
+
+## Participants
+
+<!-- for team/country-based tournaments -->
```
Comment (TPGPL) [inline wiki/Tournaments/YHC/2/en.md:56]:
```suggestion
```

File: wiki/Tournaments/YHC/2/en.md [hunk 5]
```diff
@@ -0,0 +1,322 @@
...
+
+| Team Code | Team Name | Members |
```
Comment (TPGPL) [inline wiki/Tournaments/YHC/2/en.md:58]:
```suggestion
| Team code | Team name | Members |
```

File: wiki/Tournaments/YHC/2/en.md [hunk 6]
```diff
@@ -0,0 +1,322 @@
...
+
+| Placing | Team |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | **懒人三项** |
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | **老年娱乐** |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | **已经结束啦** |
+
+## Mappools
+
+### Elimination Stage 2 (Round 3)
```
Comment (TPGPL) [inline wiki/Tournaments/YHC/2/en.md:91]:
```suggestion
### Elimination stage 2 (Round 3)
```

File: wiki/Tournaments/YHC/2/en.md [hunk 7]
```diff
@@ -0,0 +1,322 @@
...
+
+**[Download the mappack here! (122.5 MB)](https://drive.google.com/file/d/12rFoRjc-raITz7Ib55JDOA9RHvwnEOmJ/view?usp=share_link)**
+
+- HD
```
Comment (TPGPL) [inline wiki/Tournaments/YHC/2/en.md:95]:
```suggestion
- Hidden
```

File: wiki/Tournaments/YHC/2/en.md [hunk 8]
```diff
@@ -0,0 +1,322 @@
...
+  1. [Hana - Sakura no Uta (Zekk) [Subahibi's Extra]](https://osu.ppy.sh/beatmapsets/1648596#osu/3468294)
+  2. [Water Color Melody. - MotherGoose o Kikinagara (AdveNt) [Insane]](https://osu.ppy.sh/beatmapsets/1347987#osu/3113813)
+  3. [BlackY feat. Risa Yuzuki - Shuumatsu Kinema (My Angel Jeremy) [Insane]](https://osu.ppy.sh/beatmapsets/1820113#osu/3735247)
+  4. [69 de 74 - RTTR (Reywateil) [Xenon's Another]](https://osu.ppy.sh/beatmapsets/1447300#osu/3196719)
+  5. [hitorie - Little Cry Baby (moph) [Insane]](https://osu.ppy.sh/beatmapsets/1098258#osu/2381878)
+  6. [Sakamoto Maaya - Dokuhaku (Kalibe) [Insane]](https://osu.ppy.sh/beatmapsets/1426532#osu/2938608)
+  7. [Silent Siren - LOVE no Shirushi (alacat) [Insane]](https://osu.ppy.sh/beatmapsets/122444#osu/313165)
+- NM
```
Comment (TPGPL) [inline wiki/Tournaments/YHC/2/en.md:103]:
```suggestion
- NoMod
```

File: wiki/Tournaments/YHC/2/en.md [hunk 9]
```diff
@@ -0,0 +1,322 @@
...
+  1. [nowisee - microser (bilibilicnm) [Extra]](https://osu.ppy.sh/beatmapsets/1456710#osu/3017432)
+  2. [Metomate - Guren (Hey lululu) [lu^3 x lealu's Insane]](https://osu.ppy.sh/beatmapsets/1544987#osu/3157972)
+- DT
```
Comment (TPGPL) [inline wiki/Tournaments/YHC/2/en.md:106]:
```suggestion
- DoubleTime
```

File: wiki/Tournaments/YHC/2/en.md [hunk 10]
```diff
@@ -0,0 +1,322 @@
...
+  1. [Ylvis - The Fox (W h i t e) [Fox]](https://osu.ppy.sh/beatmapsets/119980#osu/307804)
+  2. [kozato - Tsuki -Yue- (Gust) [kanor's Hard]](https://osu.ppy.sh/beatmapsets/1401651#osu/2896899)
+  3. [Lana Del Rey - Summertime Sadness (piroshki) [A Sweet Kiss]](https://osu.ppy.sh/beatmapsets/1401651#osu/2896899)
+- FM
```
Comment (TPGPL) [inline wiki/Tournaments/YHC/2/en.md:110]:
```suggestion
- FreeMod
```

File: wiki/Tournaments/YHC/2/en.md [hunk 11]
```diff
@@ -0,0 +1,322 @@
...
+  1. [Charli XCX - Boom Clap (schoolboy) [Expert]](https://osu.ppy.sh/beatmapsets/730792#osu/1542345)
+  2. [Soleily feat. SUNRaiSE - Rise Above (Nytrocide_) [-Sylvari's Insane]](https://osu.ppy.sh/beatmapsets/1767895#osu/3661739)
+  3. [Tokyo Jihen - Sounan (Uberzolik) [AIR's Insane]](https://osu.ppy.sh/beatmapsets/967321#osu/2033671)
+- TB
```
Comment (TPGPL) [inline wiki/Tournaments/YHC/2/en.md:114]:
```suggestion
- Tiebreaker
```

File: wiki/Tournaments/YHC/2/en.md [hunk 12]
```diff
@@ -0,0 +1,322 @@
...
+  1. [**DystopiaGround - AugoEidEs (jonathanlfj) [Angelic]**](https://osu.ppy.sh/beatmapsets/136969#osu/342543)
```
Comment (TPGPL) [inline wiki/Tournaments/YHC/2/en.md:115]:
```suggestion
  1. **[DystopiaGround - AugoEidEs (jonathanlfj) [Angelic]](https://osu.ppy.sh/beatmapsets/136969#osu/342543)**
```

File: wiki/Tournaments/YHC/2/en.md [hunk 13]
```diff
@@ -0,0 +1,322 @@
...
+- NM
+  1. [nowisee - microser (bilibilicnm) [Extra]](https://osu.ppy.sh/beatmapsets/1456710#osu/3017432)
+  2. [Metomate - Guren (Hey lululu) [lu^3 x lealu's Insane]](https://osu.ppy.sh/beatmapsets/1544987#osu/3157972)
+- DT
+  1. [Ylvis - The Fox (W h i t e) [Fox]](https://osu.ppy.sh/beatmapsets/119980#osu/307804)
+  2. [kozato - Tsuki -Yue- (Gust) [kanor's Hard]](https://osu.ppy.sh/beatmapsets/1401651#osu/2896899)
+  3. [Lana Del Rey - Summertime Sadness (piroshki) [A Sweet Kiss]](https://osu.ppy.sh/beatmapsets/1401651#osu/2896899)
+- FM
+  1. [Charli XCX - Boom Clap (schoolboy) [Expert]](https://osu.ppy.sh/beatmapsets/730792#osu/1542345)
+  2. [Soleily feat. SUNRaiSE - Rise Above (Nytrocide_) [-Sylvari's Insane]](https://osu.ppy.sh/beatmapsets/1767895#osu/3661739)
```
Comment (TPGPL) [inline wiki/Tournaments/YHC/2/en.md:112]:
```suggestion
  2. [Soleily feat. SUNRaiSE - Rise Above (Nytrocide\_) [-Sylvari's Insane]](https://osu.ppy.sh/beatmapsets/1767895#osu/3661739)
```

File: wiki/Tournaments/YHC/2/en.md [hunk 14]
```diff
@@ -0,0 +1,322 @@
...
+  3. [Tokyo Jihen - Sounan (Uberzolik) [AIR's Insane]](https://osu.ppy.sh/beatmapsets/967321#osu/2033671)
+- TB
+  1. [**DystopiaGround - AugoEidEs (jonathanlfj) [Angelic]**](https://osu.ppy.sh/beatmapsets/136969#osu/342543)
+
+### Elimination Stage 1 (Round 2)
```
Comment (TPGPL) [inline wiki/Tournaments/YHC/2/en.md:117]:
```suggestion
### Elimination stage 1 (Round 2)
```

File: wiki/Tournaments/YHC/2/en.md [hunk 15]
```diff
@@ -0,0 +1,322 @@
...
+
+**[Download the mappack here! (121.4 MB)](https://drive.google.com/file/d/1nOrkgMX0pNqj9b-HgxLBjCc-pJF5FAEm/view?usp=share_link)**
+
+- HD
```
Comment (TPGPL) [inline wiki/Tournaments/YHC/2/en.md:121]:
```suggestion
- Hidden
```

File: wiki/Tournaments/YHC/2/en.md [hunk 16]
```diff
@@ -0,0 +1,322 @@
...
+  1. [BAND-MAID - I can’t live without you. (Lasse) [Insane]](https://osu.ppy.sh/beatmapsets/1093430#osu/2288966)
+  2. [Lime - Renai Syndrome (apaffy) [Another]](https://osu.ppy.sh/beatmapsets/166062#osu/859667)
+  3. [Sangatsu no Phantasia - Koufuku na Wagamama (Jiysea) [Insane]](https://osu.ppy.sh/beatmapsets/1478828#osu/3035501)
+  4. [M2U - Promise (feat. Sherie) (Ultima Fox) [Serenity]](https://osu.ppy.sh/beatmapsets/1005492#osu/2104403)
+  5. [Toromaru - Enigma (Avena) [EXHAUST]](https://osu.ppy.sh/beatmapsets/460380#osu/1003687)
+  6. [senya - Sono Kami no Dilemma ni (Serafeim) [Wheel of Misfortune]](https://osu.ppy.sh/beatmapsets/1391331#osu/2872872)
+- NM
```
Comment (TPGPL) [inline wiki/Tournaments/YHC/2/en.md:128]:
```suggestion
- NoMod
```

File: wiki/Tournaments/YHC/2/en.md [hunk 17]
```diff
@@ -0,0 +1,322 @@
...
+  1. [Vivy (CV: Yagi Kairi) - Sing My Pleasure (Andrea) [Collab Insane]](https://osu.ppy.sh/beatmapsets/1470500#osu/3019025)
+  2. [Run Girls, Run！ - Share the light (Amateurre) [Shizurre's Insane]](https://osu.ppy.sh/beatmapsets/1693654#osu/3461651)
+- DT
```
Comment (TPGPL) [inline wiki/Tournaments/YHC/2/en.md:131]:
```suggestion
- DoubleTime
```

File: wiki/Tournaments/YHC/2/en.md [hunk 18]
```diff
@@ -0,0 +1,322 @@
...
+  1. [senya - Hitorishizuka (Satellite) [Satellite]](https://osu.ppy.sh/beatmapsets/95839#osu/261417)
+  2. [Silver Forest Feat. Aki & Sakura Mayuko - Summer time Heaven -after (RVMathew) [Lunatic Lite]](https://osu.ppy.sh/beatmapsets/729279#osu/1554793)
+- FM
```
Comment (TPGPL) [inline wiki/Tournaments/YHC/2/en.md:134]:
```suggestion
- FreeMod
```

File: wiki/Tournaments/YHC/2/en.md [hunk 19]
```diff
@@ -0,0 +1,322 @@
...
+  1. [Akatsuki Records - Silent Silence Siren (Zelq) [Rolniczy's Insane]](https://osu.ppy.sh/beatmapsets/866443#osu/1824472)
+  2. [Good Kid - Slingshot (wafer) [Crowley's Insane]](https://osu.ppy.sh/beatmapsets/1802930#osu/3739576)
+  3. [cadode - Kaika (VINXIS) [Shadows]](https://osu.ppy.sh/beatmapsets/1754413#osu/3590290)
+- TB
```
Comment (TPGPL) [inline wiki/Tournaments/YHC/2/en.md:138]:
```suggestion
- Tiebreaker
```

File: wiki/Tournaments/YHC/2/en.md [hunk 20]
```diff
@@ -0,0 +1,322 @@
...
+  1. [**earthmind - Another Heaven (Kalibe) [Sakura]**](https://osu.ppy.sh/beatmapsets/571626#osu/1308941)
```
Comment (TPGPL) [inline wiki/Tournaments/YHC/2/en.md:139]:
```suggestion
  1. **[earthmind - Another Heaven (Kalibe) [Sakura]](https://osu.ppy.sh/beatmapsets/571626#osu/1308941)**
```

File: wiki/Tournaments/YHC/2/en.md [hunk 21]
```diff
@@ -0,0 +1,322 @@
...
+
+### Group Stage (Round 1)
```
Comment (TPGPL) [inline wiki/Tournaments/YHC/2/en.md:141]:
```suggestion
### Group stage (Round 1)
```

File: wiki/Tournaments/YHC/2/en.md [hunk 22]
```diff
@@ -0,0 +1,322 @@
...
+
+**[Download the mappack here! (103 MB)](https://drive.google.com/file/d/16EGRC4DnalO_YEiENcBLBTi9H05XcHiE/view?usp=share_link)**
+
+- HD
```
Comment (TPGPL) [inline wiki/Tournaments/YHC/2/en.md:145]:
```suggestion
- Hidden
```

File: wiki/Tournaments/YHC/2/en.md [hunk 23]
```diff
@@ -0,0 +1,322 @@
...
+  1. [Wakeshima Kanon - world's end, girl's rondo (Agatsu) [Irohas' Insane]](https://osu.ppy.sh/beatmapsets/997439#osu/2089366)
+  2. [dj TAKA - snow storm (Oracle) [Another]](https://osu.ppy.sh/beatmapsets/151033#osu/372448)
+  3. [Tsukuyomi - Dareka no Shinzou ni Nareta nara (Mononymous) [too's Insane]](https://osu.ppy.sh/beatmapsets/1544600#osu/3239199)
+  4. [:) feat. KAFU - Alma (Ducky-) [Insane]](https://osu.ppy.sh/beatmapsets/1677752#osu/3448129)
+  5. [kessoku band - Guitar to Kodoku to Aoi Hoshi (ponbot) [Ame's Insane]](https://osu.ppy.sh/beatmapsets/1880007#osu/3902695)
+  6. [DJ YOSHITAKA - Evans (Leader) [vikala's Insane]](https://osu.ppy.sh/beatmapsets/1023491#osu/2141049)
+- NM
```
Comment (TPGPL) [inline wiki/Tournaments/YHC/2/en.md:152]:
```suggestion
- NoMod
```

File: wiki/Tournaments/YHC/2/en.md [hunk 24]
```diff
@@ -0,0 +1,322 @@
...
+  1. [Limonene - (please) replay (Realazy) [(lasse's) insane]](https://osu.ppy.sh/beatmapsets/1279941#osu/2661092)
+  2. [yanaginagi - Usotsuki (Cellina) [Collab Insane]](https://osu.ppy.sh/beatmapsets/1791297#osu/3743459)
+- DT
```
Comment (TPGPL) [inline wiki/Tournaments/YHC/2/en.md:155]:
```suggestion
- DoubleTime
```

File: wiki/Tournaments/YHC/2/en.md [hunk 25]
```diff
@@ -0,0 +1,322 @@
...
+  1. [Marina and the Diamonds - Teen Idle (-Waterlemon) [Hard]](https://osu.ppy.sh/beatmapsets/1784342#osu/3675891)
+  2. [Gesu no Kiwami Otome. - Digital Mogura (dsco) [Hard]](https://osu.ppy.sh/beatmapsets/737340#osu/2587641)
+- FM
```
Comment (TPGPL) [inline wiki/Tournaments/YHC/2/en.md:158]:
```suggestion
- FreeMod
```

File: wiki/Tournaments/YHC/2/en.md [hunk 26]
```diff
@@ -0,0 +1,322 @@
...
+  1. [Silver Forest - Kero (9) destiny (Bloxi) [Kloster's Lunatic]](https://osu.ppy.sh/beatmapsets/1670776#osu/3424391)
+  2. [MindaRyn - Like Flames (MajK00) [AkagEnu's Insane]](https://osu.ppy.sh/beatmapsets/1693866#osu/3521488)
+  3. [BlackY feat. Risa Yuzuki - Stellalude (Hinsvar) [Insane]](https://osu.ppy.sh/beatmapsets/1515209#osu/3139469)
+- TB
```
Comment (TPGPL) [inline wiki/Tournaments/YHC/2/en.md:162]:
```suggestion
- Tiebreaker
```

File: wiki/Tournaments/YHC/2/en.md [hunk 27]
```diff
@@ -0,0 +1,322 @@
...
+  1. [**Marika - Truth and Light (ponbot) [gazimal’s Insane]**](https://osu.ppy.sh/beatmapsets/1594534#osu/3256904)
```
Comment (TPGPL) [inline wiki/Tournaments/YHC/2/en.md:163]:
```suggestion
  1. **[Marika - Truth and Light (ponbot) [gazimal’s Insane]](https://osu.ppy.sh/beatmapsets/1594534#osu/3256904)**
```

File: wiki/Tournaments/YHC/2/en.md [hunk 28]
```diff
@@ -0,0 +1,322 @@
...
+
+## Match results
+
+### Elimination Stage 2 (Round 3)
```
Comment (TPGPL) [inline wiki/Tournaments/YHC/2/en.md:167]:
```suggestion
### Elimination stage 2 (Round 3)
```

File: wiki/Tournaments/YHC/2/en.md [hunk 29]
```diff
@@ -0,0 +1,322 @@
...
+
+| Team 1 |  |  | Team 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| 已经结束啦 ::{ flag=CN }:: | 2 | **7** | ::{ flag=CN }:: **老年娱乐** | [#106494978](https://osu.ppy.sh/community/matches/106494978) |
+| 主播教我HD刷170 ::{ flag=CN }:: | 2 | **7** | ::{ flag=CN }:: **懒人三项** | [#106474264](https://osu.ppy.sh/community/matches/106474264) |
+| 已经结束啦 ::{ flag=CN }:: | 2 | **7** | ::{ flag=CN }:: **懒人三项** | [#106647542](https://osu.ppy.sh/community/matches/106647542) |
+| 老年娱乐 ::{ flag=CN }:: | 2 | **7** | ::{ flag=CN }:: **懒人三项** | [#106652243](https://osu.ppy.sh/community/matches/106652243) |
```
Comment (TPGPL) [inline wiki/Tournaments/YHC/2/en.md:174]:
```suggestion
| 已经结束啦 ::{ flag=CN }:: | 2 | **7** | ::{ flag=CN }:: **老年娱乐** | [#1](https://osu.ppy.sh/community/matches/106494978) |
| 主播教我HD刷170 ::{ flag=CN }:: | 2 | **7** | ::{ flag=CN }:: **懒人三项** | [#1](https://osu.ppy.sh/community/matches/106474264) |
| 已经结束啦 ::{ flag=CN }:: | 2 | **7** | ::{ flag=CN }:: **懒人三项** | [#1](https://osu.ppy.sh/community/matches/106647542) |
| 老年娱乐 ::{ flag=CN }:: | 2 | **7** | ::{ flag=CN }:: **懒人三项** | [#1](https://osu.ppy.sh/community/matches/106652243) |
```

File: wiki/Tournaments/YHC/2/en.md [hunk 30]
```diff
@@ -0,0 +1,322 @@
...
+
+### Elimination Stage 1 (Round 2)
```
Comment (TPGPL) [inline wiki/Tournaments/YHC/2/en.md:176]:
```suggestion
### Elimination stage 1 (Round 2)
```

File: wiki/Tournaments/YHC/2/en.md [hunk 31]
```diff
@@ -0,0 +1,322 @@
...
+
+| Team 1 |  |  | Team 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| 主播教我HD刷170 ::{ flag=CN }:: | 2 | **6** | ::{ flag=CN }:: **已经结束啦** | [#106265442](https://osu.ppy.sh/community/matches/106265442) |
+| **老年娱乐**::{ flag=CN }:: | **6** | 1 | ::{ flag=CN }:: 东雪莲后援会 | [#106245589](https://osu.ppy.sh/community/matches/106245589) |
+| **主播教我HD刷170** ::{ flag=CN }:: | **6** | 1 | ::{ flag=CN }:: 无所谓 帕老师会出手 | [#106267609](https://osu.ppy.sh/community/matches/106267609) |
+| 东雪莲后援会 ::{ flag=CN }:: | 1 | **6** | ::{ flag=CN }:: **懒人三项** | [#106288941](https://osu.ppy.sh/community/matches/106288941) |
```
Comment (TPGPL) [inline wiki/Tournaments/YHC/2/en.md:183]:
```suggestion
| 主播教我HD刷170 ::{ flag=CN }:: | 2 | **6** | ::{ flag=CN }:: **已经结束啦** | [#1](https://osu.ppy.sh/community/matches/106265442) |
| **老年娱乐**::{ flag=CN }:: | **6** | 1 | ::{ flag=CN }:: 东雪莲后援会 | [#1](https://osu.ppy.sh/community/matches/106245589) |
| **主播教我HD刷170** ::{ flag=CN }:: | **6** | 1 | ::{ flag=CN }:: 无所谓 帕老师会出手 | [#1](https://osu.ppy.sh/community/matches/106267609) |
| 东雪莲后援会 ::{ flag=CN }:: | 1 | **6** | ::{ flag=CN }:: **懒人三项** | [#1](https://osu.ppy.sh/community/matches/106288941) |
```

File: wiki/Tournaments/YHC/2/en.md [hunk 32]
```diff
@@ -0,0 +1,322 @@
...
+
+### Group Stage (Round 1)
```
Comment (TPGPL) [inline wiki/Tournaments/YHC/2/en.md:185]:
```suggestion
### Group stage (Round 1)
```

File: wiki/Tournaments/YHC/2/en.md [hunk 33]
```diff
@@ -0,0 +1,322 @@
...
+
+#### Group A
+
+| Team 1 |  |  | Team 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| 无所谓 帕老师会出手 ::{ flag=CN}:: | 1 | **5** | ::{ flag=CN }:: **主播教我HD刷170** | [#105951662](https://osu.ppy.sh/community/matches/105951662) |
+| **主播教我HD刷170** ::{ flag=CN }:: | **5** | 2 | ::{ flag=CN }:: 东雪莲后援会 | [#105992915](https://osu.ppy.sh/community/matches/105992915) |
+| **东雪莲后援会** ::{ flag=CN }:: | **5** | 1 | ::{ flag=CN }:: 无所谓 帕老师会出手 | [#106089411](https://osu.ppy.sh/community/matches/106089411) |
```
Comment (TPGPL) [inline wiki/Tournaments/YHC/2/en.md:193]:
```suggestion
| 无所谓 帕老师会出手 ::{ flag=CN}:: | 1 | **5** | ::{ flag=CN }:: **主播教我HD刷170** | [#1](https://osu.ppy.sh/community/matches/105951662) |
| **主播教我HD刷170** ::{ flag=CN }:: | **5** | 2 | ::{ flag=CN }:: 东雪莲后援会 | [#1](https://osu.ppy.sh/community/matches/105992915) |
| **东雪莲后援会** ::{ flag=CN }:: | **5** | 1 | ::{ flag=CN }:: 无所谓 帕老师会出手 | [#1](https://osu.ppy.sh/community/matches/106089411) |
```

File: wiki/Tournaments/YHC/2/en.md [hunk 34]
```diff
@@ -0,0 +1,322 @@
...
+
+#### Group B
+
+| Team 1 |  |  | Team 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| 老年娱乐 ::{ flag=CN }:: | **5** | 1 | ::{ flag=CN }:: 已经结束啦 | [#105975299](https://osu.ppy.sh/community/matches/105975299) |
+| 已经结束啦 ::{ flag=CN }:: | **5** | 1 | ::{ flag=CN }:: 懒人三项 | [#105990432](https://osu.ppy.sh/community/matches/105990432) |
+| 懒人三项 ::{ flag=CN }:: | 4 | **5** | ::{ flag=CN }:: 老年娱乐 | [#106113040](https://osu.ppy.sh/community/matches/106113040) |
```
Comment (TPGPL) [inline wiki/Tournaments/YHC/2/en.md:201]:
```suggestion
| **老年娱乐** ::{ flag=CN }:: | **5** | 1 | ::{ flag=CN }:: 已经结束啦 | [#1](https://osu.ppy.sh/community/matches/105975299) |
| **已经结束啦** ::{ flag=CN }:: | **5** | 1 | ::{ flag=CN }:: 懒人三项 | [#1](https://osu.ppy.sh/community/matches/105990432) |
| **懒人三项** ::{ flag=CN }:: | 4 | **5** | ::{ flag=CN }:: 老年娱乐 | [#1](https://osu.ppy.sh/community/matches/106113040) |
```

File: wiki/Tournaments/YHC/2/en.md [hunk 35]
```diff
@@ -0,0 +1,322 @@
...
+    - Exclusion of specific players for one beatmap.
+    - Exclusion of specific players for an entire match.
+    - Declaring the match as forfeited, or as a win by default for the other team.
+    - Disqualification from the entire tournament.
+    - Disqualification from the current and future official tournaments, until appealed.
+16. Referees may allow, at their discretion, lower or higher tolerances for timers.
+17. The tournament managers may request liveplays or recordings of individual players or teams at any point in the tournament without prior warning.
+18. The tournament managers reserve the right to modify these rules at any moment. Any such changes will be announced in advance.
+
+### Tournament Registration
```
Comment (TPGPL) [inline wiki/Tournaments/YHC/2/en.md:244]:
```suggestion
### Tournament registration
```

File: wiki/Tournaments/YHC/2/en.md [hunk 36]
```diff
@@ -0,0 +1,322 @@
...
+
+1. A player who meets the registration conditions can register through the [Registration Form](https://docs.qq.com/form/page/DUm5TSlRpTG11WHdx). Registration will close on 2022-12-25.
+2. Registration Conditions:
+   - pp limitation: 2000pp - 4500pp
+3. If a player registered successfully, a **cost** will be calculated and will affect the team-up stage.
+
+### Team-up Information
```
Comment (TPGPL) [inline wiki/Tournaments/YHC/2/en.md:251]:
```suggestion
### Team-up information
```

**Review by TPGPL — CHANGES_REQUESTED** [review]
apply the changes to ZH where possible as well

**Comment by YumeMuzi** [comment]
> pretty sure wiki doesn't support `.svg` files like that, you should convert them to `.jpg` and add some background as it may not be well visible on the site

ok i'll replace these svg files tmr

**Comment by YumeMuzi** [comment]
should I wait for #8685 to be merged? we edited the same file but in different paragraphs.

**Comment by TPGPL** [comment]
doesn't matter i think, both can be merged in whatever order

**Review by TPGPL — APPROVED** [review]

**Pull request merged**